### PR TITLE
fix: feed quick actions and search bar

### PR DESCRIPTION
**Pull request by imrizwan**
- feed fixed
- search bar fixed
- quick actions fixed
on native side

### CODE REVIEW

**Review by andrew-bierman — APPROVED** [review]
lgtm

**Review by andrew-bierman — CHANGES_REQUESTED** [review]
see comments, this is almost ready to go though

File: packages/app/components/pack/PackDetails.tsx [hunk 1]
```diff
@@ -20,8 +20,17 @@ import { useUserPacks } from 'app/hooks/packs/useUserPacks';
 import { useFetchSinglePack } from '../../hooks/packs';
 
 export function PackDetails() {
-  const searchParams = new URLSearchParams(this.location.search);
-  const canCopy = searchParams.get('copy');
+
+  const getParams = () => {
+    if (Platform.OS === 'web') {
+      const searchParams = new URLSearchParams(this.location.search);
```
[the remark this answers is not in the document]
Comment (imrizwan) [reply packages/app/components/pack/PackDetails.tsx:29]: Changed to solito for cross platform

File: packages/app/components/user/UserDataContainer.tsx [hunk 2]
```diff
@@ -125,7 +126,7 @@ export default function UserDataContainer({
             //     />
             //   ))
             // )
-            <FlatList
+            Platform.OS === 'web' ? <FlatList
```
[the remark this answers is not in the document]
Comment (imrizwan) [reply packages/app/components/user/UserDataContainer.tsx:129]: Converted flatlist to virtualized list, and made a different structure to remove virtualized list error

**Comment by imrizwan** [comment]
> see comments, this is almost ready to go though

Comments resolved

**Comment by andrew-bierman** [comment]
pack details on web is blank for me, might just be my next environment, looking into it

**Comment by andrew-bierman** [comment]
pack details still blank on my side, are you noticing the same ?

**Comment by imrizwan** [comment]
> pack details still blank on my side, are you noticing the same ?

I am receiving this on pack details page
image

**Comment by imrizwan** [comment]
> pack details still blank on my side, are you noticing the same ?

 - No, Seems to be working fine, but let me check if there's any problem

**Review by andrew-bierman — APPROVED** [review]

**Pull request merged**